src/open_inwoner/openklant/cms_plugins.py

```diff
@@ -8,16 +8,19 @@
 
 from open_inwoner.ckeditor5.widgets import CKEditorWidget
 
-#
-# contact form plugin
-#
-
 
 class ContactFormConfig(CMSPlugin):
-    description = models.TextField(
-        _("Description"),
+    description_auth = models.TextField(
```

Comment:
I am happy for more verbose names:

```suggestion
    description_authenticated_user = models.TextField(
```